src/dex-ui/pages/Governance/ManageVotingPower/ManageGovToken.tsx

```diff
@@ -0,0 +1,195 @@
+          <Tab>Lock</Tab>
+          <Tab>Unlock</Tab>
+        </TabList>
+        <TabPanels>
+          <TabPanel>
+            <FormInput<"lockAmount">
+              inputProps={{
+                id: "lockAmount",
+                pointerEvents: "all",
+                label: <InputLabel balance="20000.00" />,
+                type: "number",
+                unit: (
+                  <RightUnitItem
+                    handleHalfButtonClicked={handleHalfButtonClicked}
+                    handleMaxButtonClicked={handleMaxButtonClicked}
+                  />
+                ),
+                placeholder: "",
+                register: {
+                  ...props.form.register("lockAmount", {
+                    required: { value: true, message: "Amount cannot be more than available balance." },
+                  }),
+                },
+              }}
+              isInvalid={Boolean(props.form.formState.errors.lockAmount)}
+              errorMessage={props.errors.lockAmount && props.errors.lockAmount.message}
+            />
+          </TabPanel>
+          <TabPanel>
+            <FormInput<"unLockAmount">
+              inputProps={{
+                id: "unLockAmount",
+                pointerEvents: "all",
+                label: <InputLabel balance="10000.00" />,
+                type: "number",
+                unit: (
+                  <RightUnitItem
+                    handleHalfButtonClicked={handleHalfButtonClicked}
+                    handleMaxButtonClicked={handleMaxButtonClicked}
+                  />
+                ),
+                placeholder: "",
+                register: {
+                  ...props.form.register("unLockAmount", {
+                    required: { value: true, message: "unlock amount cannot be more than locked balance." },
+                  }),
+                },
+              }}
+              isInvalid={Boolean(props.form.formState.errors.unLockAmount)}
+              errorMessage={props.errors.unLockAmount && props.errors.unLockAmount.message}
+            />
+          </TabPanel>
+        </TabPanels>
+      </Tabs>
+      <Button
+        variant="primary"
+        type="submit"
+        isDisabled={isSubmitButtonDisabled}
+        leftIcon={tabIndex === 0 ? <LockIcon /> : <UnlockIcon />}
+        onClick={props.form.handleSubmit(props.onSubmitHandle)}
+        margin="0px 15px 0px 15px"
+      >
+        {tabIndex === 0 ? "Lock" : "Unlock"}
+      </Button>
+    </Flex>
+  );
+}
+
+export const ManageGovToken = (props: ManageGovTokenProps) => {
+  const [dialogsOpenState, setDialogsOpenState] = useState(false);
+
+  const manageGovTokenForm = useForm<InputTokenAmountData>({
+    defaultValues: {
+      unLockAmount: "",
+      lockAmount: "",
+    },
+  });
+  manageGovTokenForm.watch(["lockAmount", "unLockAmount"]);
+
+  function onSubmit(data: InputTokenAmountData) {
+    console.log("Submit Pressed", data);
+  }
+
+  return (
+    <form>
+      <AlertDialog
```

Comment:
Here, `Loading Dialog`, `Success Dialog` and `Error Dialog` will also come that change will come once we get the SC for this.